src/components/helpers.js

```diff
@@ -155,6 +155,10 @@ export const normalizeTransferHistoryMessage = (transferData, sender) => {
 		},
 		department: () => {
 			const to = nextDepartment && nextDepartment.name;
+			if (!sender.username) {
+				const from = 'the agent';
```

Comment:
We'd have to handle translations for this string. I'd suggest adding a new translation string altogether, in-case agent info is hidden